lib/ffmpeg/transcoder.rb

```diff
@@ -170,8 +173,12 @@ def delete_files(destination)
 
     def transcode_movie
       pre_encode_if_necessary
+      temp_output_dir = "#{TEMP_DIR}/output/"
```

Comment:
Retain prior whitespace
```suggestion
      
      temp_output_dir = "#{TEMP_DIR}/output/"
```